widget/toolbar.go

```diff
@@ -23,9 +23,9 @@ type ToolbarAction struct {
 
 // ToolbarObject gets a button to render this ToolbarAction
 func (t *ToolbarAction) ToolbarObject() fyne.CanvasObject {
-	button := NewButtonWithIcon("", t.Icon, t.OnActivated)
-	button.HideShadow = true
-
+	//button := NewButtonWithIcon("", t.Icon, t.OnActivated)
```

Comment:
Please don't commit commented out code - we have version control to track changes.

Reply:
I know. I just missed it while cleaning up.